app/models/study.rb

```diff
@@ -968,7 +968,8 @@ def search_weight(terms)
         terms: {}
     }
     terms.each do |term|
-      text_blob = "#{self.name} #{self.description}"
+      author_names = authors.pluck(:first_name, :last_name).flatten.join(' ')
+      text_blob = "#{self.name} #{self.description} #{author_names}"
```

Comment:
If we're indexing `institution` (from `app/models/author.rb#L35`), then we also need to add it here, otherwise the end-to-end integration on matching won't work correctly.